tutorials/cheyenne/launch_ensemble_batch.py

```diff
@@ -0,0 +1,51 @@
+from smartsim import Experiment
```

Comment:
It's a little annoying, but it would be easier in the directions to turn this into a proper function that accepts a single input argument which is the Account name.